openshift/templates/app/deploy.yaml

```diff
@@ -143,7 +86,7 @@ objects:
           containers:
             - name: ${APP_NAME}-${ROLE_NAME}
               image: >-
-                image-registry.openshift-image-registry.svc:5000/${PROJECT_NAMESPACE}-tools/${APP_NAME}-${ROLE_NAME}-${IMAGE_DEPLOY}@sha256:69c2e98c46bf9b4e8eca8344e7c3627a464df969046a2ff3fc83cf781fc0c930
+                image-registry.openshift-image-registry.svc:5000/${PROJECT_NAMESPACE}-tools/${APP_NAME}-${ROLE_NAME}-${IMAGE_DEPLOY}@sha256:8cc727d67aa089c2dfe88c4f8052a27ad32ba1e5f3a49d147b588211e7b53b64
```

Comment:
I would probably remove the @sha256:etc etc etc from this file. OpenShift will autogenerate and overwrite this on every successful deploy.

Reply:
i tried this, but it didn't seem to work until i added it

Reply:
Interesting. If the sha is incorrect (or worse an old version out of date), will it fail/grab that older version?